pkg/handlers/resource.go

```diff
@@ -70,7 +70,7 @@ func (h resourceHandler) Patch(w http.ResponseWriter, r *http.Request) {
 		func() (interface{}, *errors.ServiceError) {
 			ctx := r.Context()
 			id := mux.Vars(r)["id"]
-			manifest, err := presenters.ConvertResourceManifest(patch.Manifest)
+			manifest, err := presenters.ConvertResourceManifest(patch.Manifest, patch.DeleteOption, patch.UpdateStrategy)
```

Comment:
I think we should support update these options.

Reply:
Do you mean `PATCH` these options? if so, it was covered. I am working on adding test cases for that.